src/ontology/mondo-edit.obo

```diff
@@ -519289,6 +519297,34 @@ property_value: IAO:0000231 OMO:0001000 {source="MONDO:excludeTrait"}
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/7616" xsd:anyURI
 is_obsolete: true
 
+[Term]
+id: MONDO:0100569
+name: ACD-related short telomere syndrome
+def: "A spectrum of conditions, including dyskeratosis congenita, Hoyeraal-Hreidarsson syndrome, hereditary aplastic anemia, and pulmonary fibrosis, typically characterized by shortened telomeres due to a pathogenic variant(s) in ACD that results in impaired telomere maintenance." [https://orcid.org/0009-0007-0138-2054]
+is_a: MONDO:0800469 {source="https://orcid.org/0009-0007-0138-2054"} ! ACD-related telomere biology disorder
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/25070 {source="https://orcid.org/0009-0007-0138-2054"} ! ACD
+property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0001-5208-3432
```

Comment:
Is this property still being added for new classes?
- It's not in the documentation for adding a new term https://mondo.readthedocs.io/en/stable/editors-guide/add-new-terms/#add-a-new-term
- It was not added for the new MONDO:0800469, the parent of this term

Reply:
added via protege